DEMO.md

```diff
@@ -118,17 +124,70 @@ timestamping through `bitcoindsim`:
     ```shell
     docker logs -f babylondnode0
     ```
-  - Install babylond for your architecture (Needed for local CLI access). Currently, Linux (x86_64) and Mac M1 (arm64)
-pre-built binaries are provided:
-    ```shell
-    mkdir -p $HOME/bin && gunzip -c ./babylon-private/babylond-$(uname -m).gz >$HOME/bin/babylond && chmod +x $HOME/bin/babylond
-    ```
-  - Set PATH environment variable:
-    ```shell
-    export PATH=$HOME/bin:$PATH
-    type babylond
-    babylond version
-    ```
+#### Install Babylon Binary
+
+**1.Navigate to Your Project Directory**
+Ensure you're in the root directory of your existing project where you want to add Babylon as a submodule.
+
+**2. Add Babylon as a Submodule.**
+
+```shell
+git submodule add https://github.com/babylonchain/babylon.git path/to/babylon
+git submodule update --init --recursive
+```
+
+Replace `path/to/babylon` with the desired location within your project where you want the Babylon submodule to reside.
```

Comment:
You can just list the command to switch to the right directory:
`cd external/babylon`